monkey/monkey_island/cc/ui/src/components/pages/ReportPage.js

```diff
@@ -67,6 +68,13 @@ class ReportPageComponent extends AuthComponent {
             ransomwareReport: res
           });
         });
+      this.authFetch('/api/telemetry?telem_category=file_encryption')
+        .then(res => res.json())
+        .then(res => {
+          this.setState({
+            ransomwareTelemetry: res
+          });
```

Comment:
Based on [earlier disscussion](https://github.com/guardicore/monkey/pull/1360#discussion_r676520531) this will cause a re-render on all child components(all of the reports) when only the table needs to re-render?